spec/factories/submissions.rb

```diff
@@ -24,7 +24,7 @@
     end
 
     trait :with_augmented_dissertation_file do
-      augmented_dissertation_file { Rack::Test::UploadedFile.new(file_fixture('augmented_dissertation.pdf'), 'application/pdf') }
+      augmented_dissertation_file { Rack::Test::UploadedFile.new(file_fixture('dissertation-augmented.pdf'), 'application/pdf') }
```

Comment:
The augmented PDF name is canonically the original filename with `-augmented` appended before the file extension.